src/test/ui/typeck/typeck_type_placeholder_item.stderr

```diff
@@ -603,7 +612,10 @@ error[E0121]: the type placeholder `_` is not allowed within types on item signa
   --> $DIR/typeck_type_placeholder_item.rs:205:14
    |
 LL |     const C: _;
-   |              ^ not allowed in type signatures
+   |              ^
+   |              |
+   |              not allowed in type signatures
+   |              help: provide a type for the item: `C: [type error]`
```

Comment:
Perhaps you've already noticed this, but just want to note that this is a regression.

Reply:
Looking at the code, I am not sure how these happened 🤔

Reply:
It's because I didn't bless the tests 😅